packages/e2e-test-utils/src/create-user.js

```diff
@@ -10,14 +10,21 @@ import { switchUserToAdmin } from './switch-user-to-admin';
 import { switchUserToTest } from './switch-user-to-test';
 import { visitAdminPage } from './visit-admin-page';
 
+/**
+ * @typedef {'Subscriber' | 'Contributor' | 'Author' | 'Editor' | 'Administrator'} Role
+ */
+
 /**
  * Create a new user account.
  *
  * @param {string}  username  User name.
  * @param {string?} firstName First name.
- * @param {string?} lastName  Larst name.
+ * @param {string?} lastName  Last name.
+ * @param {string?} role      Role.
+ *
+ * @return {string} Password for the newly created user account
  */
-export async function createUser( username, firstName, lastName ) {
+export async function createUser( username, firstName, lastName, role ) {
```

Comment:
We should probably change the function signature to move the non-username arguments into an object:

```suggestion
export async function createUser( username, { firstName, lastName, role } = {} ) {
```